gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.9.2-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.9.3-bin.zip
```

Comment:
Are we able to exclude these gradle wrapper updates and keep the existing version, or is it tied to another aspect of this PR that I am missing?

Reply:
Nope, it's just bumping gradle to the latest patch version is one of the first things I do when working on deprecation warnings.

Reply:
Gotcha. If it's not a blocker, could you exclude the files related to gradle build version upgrade, and keep this PR just for the deprecation-related fixes? Thanks!

Reply:
Excluded